unitTests/controllers/CategoryFacetQueryControlllerTest.ts

```diff
@@ -54,7 +55,7 @@ export function CategoryFacetQueryControllerTest() {
         const searchResults = await categoryFacetController.searchFacetValues('some value', 15);
 
         for (let i = 0; i < searchResults.length - 1; i++) {
-          expect(searchResults[i].numberOfResults).toBeGreaterThanOrEqual(searchResults[i + 1].numberOfResults);
+          expect(searchResults[i].numberOfResults).not.toBeLessThan(searchResults[i + 1].numberOfResults);
```

Comment:
Hmm, is there a difference between `toBeGreaterThanOrEqual` and `not.toBeLessThan` or just stylistic preference?

Reply:
It has to do with the fact that when we run those tests locally in a "real" browser it's using a slightly different version of jasmine which does not have that spec.